dev-packages/e2e-tests/test-applications/react-router-6/tests/transactions.test.ts

```diff
@@ -54,3 +54,45 @@ test('sends a navigation transaction with a parameterized URL', async ({ page })
     },
   });
 });
+
+test('sends an INP span', async ({ page }) => {
+  const inpSpanPromise = waitForEnvelopeItem('react-router-6', item => {
+    return item[0].type === 'span';
+  });
+
+  await page.goto(`/`);
+
+  await page.click('#exception-button');
+
+  await page.waitForTimeout(500);
+
+  // Page hide to trigger INP
+  await page.evaluate(() => {
+    window.dispatchEvent(new Event('pagehide'));
+  });
+
+  const inpSpan = await inpSpanPromise;
+
+  expect(inpSpan[1]).toEqual({
+    data: {
+      'sentry.origin': 'auto.http.browser.inp',
+      'sentry.op': 'ui.interaction.click',
+      release: 'e2e-test',
+      environment: 'qa',
+      transaction: '/',
+      'sentry.exclusive_time': expect.any(Number),
+      replay_id: expect.any(String),
+    },
+    description: 'body > div#root > input#exception-button[type="button"]',
+    op: 'ui.interaction.click',
+    parent_span_id: expect.any(String),
```

Comment:
shouldn't parent span be undefined for INP spans?

Reply:
should it? 😅 well today it is not! if there is an ongoing transaction, it will be a child of that. is that not correct? for me this _seems_ like it makes sense, but 🤷 😅

Reply:
I was under the impression that INP spans were always root spans, but maybe I'm mistaken.

Reply:
cc @Lms24 when you're eventually back, you checked this out at the time. I'll merge this PR (as it shows the current behavior) but we can double check if this is fine.